app/models/manager_refresh/inventory/middleware_manager.rb

```diff
@@ -0,0 +1,64 @@
+module ManagerRefresh::Inventory::MiddlewareManager
+  extend ActiveSupport::Concern
+  include ::ManagerRefresh::Inventory::Core # if full name is not specified, Travis raises errors.
```

Comment:
Indeed, it also works for me locally. But Travis consistently rejected it without the FQNN. You can see the error here: https://travis-ci.org/israel-hdez/manageiq/jobs/218321562#L609
This was driving me crazy. Then, I placed `print`s everywhere to see what is Travis doing and traced the error back to this.